<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDoubleSpinBox" name="kappaBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>60</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Kappa</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filenameEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>600</y>
      <width>291</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Filename</string>
    </property>
   </widget>
   <widget class="QToolButton" name="fileButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>600</y>
      <width>26</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="kappaPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="lambdaPlot" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="etaPlot" native="true">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>160</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>90</y>
      <width>671</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Time</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Start Time</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="startTimeBox">
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>End Time</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="endTimeBox">
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
         <property name="maximum">
          <double>0.000000000000000</double>
         </property>
         <property name="value">
          <double>-10.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Delta t</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="deltaTBox">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>-0.001000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>-0.001000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>600</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="lambda3Plot" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="lambda4Plot" native="true">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>380</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="lambda2Plot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>60</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Lambda</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="lambdaBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>60</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Eta</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="etaBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Lambda3</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="lambda3Box">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="lambda4Box">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Lambda4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Lambda2</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="lambda2Box">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
